frontend/src/components/ProfileContent.tsx

```diff
@@ -1,29 +1,41 @@
 import * as React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
-import { Grid, Avatar, Typography } from '@material-ui/core';
+import { useProfile } from '../utils/profile'
+import { Grid, Avatar, Typography, TextField, Button } from '@material-ui/core';
 
 // This is what the ProfileContent component expects to receive from storage.
 export type ProfileContentProps = {
-    username: string;
-    name: string;
-    profileImg: string;
-    gender: string;
-    religion: string;
-    location: string;
-    occupations: string[];
-    age: number | string,
-    interests: string[];
-};
+    name: string
+    username: string
+    age: number
+    gender: string
```

Comment:
I don't know how to change this, but I think that 'string' is too loose a type here. We only have three string options: Male, Female, and Other. We can't write 'Bozo' for instance and get anywhere.

Reply:
Isn't the string type, the type it should be when it is pulled from the database? Also, although it is set as a string, won't the user have the options to change from either Male, Female or Other without having to type in any information?

Reply:
I don't know TypeScript well enough to say what is should be, but possibly an enum. Leaving it as just a string is too loose.